src/utils/editor-loader.js

```diff
@@ -107,9 +107,13 @@ async function loadAssets(
 	html,
 	{ allowedPackages = [], disallowedPackages = [] } = {}
 ) {
-	const excludedScriptIDs = new RegExp(
-		disallowedPackages.map( ( script ) => `wp-${ script }-js` ).join( '|' )
-	);
+	const excludedScriptIDs = disallowedPackages.length
+		? new RegExp(
+				disallowedPackages
+					.map( ( script ) => `wp-${ script }-js` )
+					.join( '|' )
+		  )
+		: null;
```

Comment:
This fixes a hidden bug where an empty `disallowedPackages` value resulted in _disallowing all packages_ due to the empty regex created. Falling back to `null` avoids this.